scripts/efs-cron-pod.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: inspect-pvc-pod
+spec:
+  containers:
+  - name: inspect-pvc-container
+    image: busybox
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        set -eux
+
+        # EFS_PERSIST_MNT is the mount point for the PVC
+        EFS_PERSIST_MNT="/mnt/efs-persist"
+        EFS_HOME="$EFS_PERSIST_MNT/home"
+
+        # Generate the size report
+        du -sh $EFS_HOME/* | awk '{print $1, $2}' > $EFS_HOME/size_report.txt
+
+        # Display the size report
+        cat "$EFS_HOME/size_report.txt"
+        
+        # Keep the container running
+        sleep 3600
+    volumeMounts:
+    - name: efs-storage
+      mountPath: /mnt/efs-persist
```

Comment:
ATM we are transferring 10TB but good amount of space is occupied by a few users, e.g. the largest (500G) is by one user having 1 single NWB file.  We could/should quickly inspect and reach out asking if could be removed.

But ideally we need a simple script which may be prints N largest files per each user and sorts users by the total size or smth like that....

```
# cd /mnt/efs-persist/home; for u in *; do biggies=$(find $u -size +1000000 -print0 | xargs -0 -r ls -Sl | head -n 5 | tr '\n' '>'); [ -z "$biggies" ] || { du -sm $u; echo  "${biggies}" | tr '>' '\n'; }; done
```

but ran on the original efs

here is the pod definition we used for pod to mount target EFS

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: sleep-tender
spec:
  containers:
  - name: inspect-pvc-container
    image: busybox
    command:
        - /bin/sh
        - -c
        - ls -la /mnt/efs-persist; echo sleeping; sleep 3600; echo exiting
    volumeMounts:
    - name: efs-storage
      mountPath: /mnt/efs-persist
      readOnly: true
  volumes:
  - name: efs-storage
    persistentVolumeClaim:
      claimName: efs-persist
```